index.js

```diff
@@ -1,6 +1,30 @@
 
 import { NativeModules } from 'react-native';
+import { Buffer } from 'buffer';
 
 const RNUportSigner = NativeModules.RNUportSignerModule;
 const RNUportHDSigner = NativeModules.RNUportHDSignerModule;
-export { RNUportSigner, RNUportHDSigner };
+const DerivationPathRoot = RNUportHDSigner.UPORT_ROOT_DERIVATION_PATH;
+
+function getSignerForAddress(address, derivationPath = DerivationPathRoot, userPrompt = "") {
```

Comment:
I think we should rename the `address` field here to `handle` or `seedAlias` since that is the way it is actually used and may not be an address in the future.

The method name could also use a rename to `getSignerForHDPath` or something along those lines.
"forAddress" suggests that the address will sign and that is only true if the path is the uport root path.